src/main/scripts/preinstall.sh

```diff
@@ -1,8 +1,30 @@
 #!/usr/bin/env bash
 
```

Comment:
```suggestion
set -o errexit
set -o nounset
set -o pipefail
```

Reply:
I generally like those in the header of every shell/bash script - but nothing major